pkgs/development/python-modules/apache-airflow/default.nix

```diff
@@ -65,17 +71,24 @@
 , mkYarnPackage
 }:
 let
-  version = "2.3.3";
+  version = "2.3.4";
 
   airflow-src = fetchFromGitHub rec {
     owner = "apache";
     repo = "airflow";
     rev = "refs/tags/${version}";
-    sha256 = "sha256-N+6ljfSo6+UvSAnvDav6G0S49JZ1VJwxmaiKPV3/DjA=";
+    # Required because the GitHub archive tarballs don't appear to include tests
+    leaveDotGit = true;
```

Comment:
There might be a better way to do this, it's confusing to me why the release tarballs don't include tests even on GitHub (I'm familiar with people doing that for PyPI).